Actions/GetArtifactsForDeployment/README.md

```diff
@@ -0,0 +1,24 @@
+# Get artifacts for deployment
```

Comment:
The "for deployment" part seems unnecessary.

Reply:
actually, this action is only used to get artifacts for deployment and it is also the name of the action.

Reply:
What about delivery?
Shouldn't such action be used when delivering artifacts?

Besides, the action shouldn't be aware of how/where it's used. And all the action does is downloading artifacts for a version in a specified folder.

Reply:
We don't really have a workflow called Deliver app, where you specify a build version or like to deliver.
Delivery is always only done continuously and for that, this action is not used.